### PR TITLE
Past executives page

### PR DESCRIPTION
**Pull request by milolucero**
Adds the Past Executives page with search functionality,

I considered a table layout as a better way to display this information compared to the previous accordion per term. I also developed a search bar to filter the displayed results. 

I considered removing the "Member at Large" from previous terms, but didn't feel comfortable removing names of whom I didn't know their contribution.

Maybe this section could have a different name, such as "Previous collaborators" or "Past team members", so that it is more clear that it does not only refer to the executive team. 

Feel free to make changes or let me know if there is anything you would like me to modify.

### CODE REVIEW

**Comment by bandrson** [comment]
Looking good. Perhaps we should reach out to the current leadership team to pin down some criteria for making it onto this list as an "executive".

File: src/pages/past-executives/index.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+export default function PastExecutivesPage() {
+  const [allMembers, setAllMembers] = useState([pastExecutives]);
+  const [displayedMembers, setDisplayedMembers] = useState(pastExecutives);
+
+  // Sorts past executives to show the most recent first.
+  const sortPastExecutives = (pastExecutives) => {
+    let sortedPastExecutives = pastExecutives.toSorted((a, b) => {
+      // We will compare year and term to sort the past executives. To do this, we assign a value to each term.
+      const termOrder = {
```
Comment (bandrson) [inline src/pages/past-executives/index.js:14]: Unless I'm mistaken, there's only 3 terms. Winter, spring, and fall.

**Review by bandrson — CHANGES_REQUESTED** [review]

File: src/pages/past-executives/index.js (hunk 1 above)
Comment (milolucero) [reply src/pages/past-executives/index.js:14]: Oh my bad! I've removed the summer term option.

**Comment by milolucero** [comment]
Agreed, maybe we can hand the question to the current leadership so that they can decide how it should be handled. For now I have solved the issue with the terms and the PR is ready to merge after you review and accept the requested change.

**Review by bandrson — APPROVED** [review]
Issue resolved. We can merge for now and I'll reach out to the leadership.

**Pull request merged**